src/rom.asm

```diff
@@ -228,6 +230,19 @@ not_mmc1:
   consider_mapper(11)
   consider_mapper(30)
   consider_mapper(31)
+// consider mapper 34 only with single CHR ROM page (BNROM)
```

Comment:
BNROM has 0 CHR ROM pages (it has only CHR RAM), as you correctly check below. (It's a little confusing because the address is still stored in `chrrom_start`.)